app/actions/actionTypes.js

```diff
@@ -1,2 +1,5 @@
 export const LOAD_USER_SUCCESS = 'LOAD_USER_SUCCESS';
 export const GET_USER_SUCCESS = 'GET_USER_SUCCESS';
+export const CREATE_USER_SUCCESS = 'CREATE_USER_SUCCESS';
+export const UPDATE_USER_SUCCESS = 'UPDATE_USER_SUCCESS';
+export const DELETE_USER_SUCCESS = 'DELETE_USER_SUCCESS';
```

Comment:
Sounds good! To be honest, it still sounds a little bit weird to me -- but being that it is a framework convention, as you said, I think it's safest to not go against the current.

Sorry for the misguided questions, you can tell I'm learning some of this stuff as well. :) I hope it's ok.